src/content/docs/journey-large-logs/filter.mdx

```diff
@@ -0,0 +1,143 @@
+
+## Decide which logs to drop [#decide]
+
+Deciding which logs to keep and which logs to drop is a highly specific decision for each team and organization. Logs valuable to one organization may not be valuable to another. Regardless, here are a few suggestions on how to decide which logs are valuable and which to drop:
+
+* **What logs do your team already use**: If your team already manually reviews a subset of logs regularly, that indicates those logs are valuable and should not be dropped. Likewise, if there is a set of lgos your team never looks at that might indicate they should be dropped.
+* **What apps and systems are producing the most logs**: an app or system that creates a large amount of logs indicates you should spend time deciding what to do with those logs. Is it a valuable and widely used app which indicates you should keep most of the logs? Is it a redundent system which is spewing logs with minimal value? 
+
+Do take note that while an app or system may be rarely used, that doesn't mean logs are always not valuable. You would hate to drop logs from an application that is barely used only for that application to go down in a few months with no easy way to troubleshoot. 
+
+<img
+  title="Logs architecture for drop filters in New Relic"
+  alt="Diagram of logs architecture for drop filters in New Relic"
+  src={logsIngestPipeline}
+/>
+
+<figcaption>
+  During the ingestion process, customer log data can be parsed, transformed, or dropped before being stored in the New Relic database (NRDB).
+</figcaption>
+
+## Filter your log ingest [#filter-steps]
+
+The following steps will guide you through how to drop logs in the New Relic UI.
+
+
+Let's say Acme Corp creates 2TB of logs each day. They decide this too many logs to ingest for both cost and usability reasons. They take a look at their logs and realize over half of their daily logs are from a legacy Node.js application. While they need to keep the app around, they don't care for the logs created by this app. They decided to drop all logs created from the Node app.
+
+
+<Steps>
+    <Step>
+        ### Navigate to the UI
+
+        Go to **[one.newrelic.com > Logs](https://one.newrelic.com/launcher/logger.log-launcher)**
+    </Step>
+    <Step>
+        ### Create your drop rule
+
+        Filter or query to the specific set of logs that contain the data to be dropped.
+
+        There are a few ways to do this, but the easiest is to query for the logs you want to drop. In this case, ACME Corp would do the following:
+
+
+<SideBySide>
+    <Side>
+
+        1. Select **All partitions** near the search bar
+        2. Enter their query. In this case `logtype=node` 
+        3. Press enter and confirm the correct logs appear. 
+        4. Once the query is active, from **Manage data** on the left nav of the Logs UI, click **Create drop filter**.
```

Comment:
`Once the query is active, from Manage data on the left nav of the Logs UI, click Create drop filter.`

this is tough to parse